_posts/2018-04-20-totw-24.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Tip of the Week #24: Copies, Abbrv."
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/24
+type: markdown
+order: "024"
+---
+
+Originally posted as TotW #24 on Nov 26, 2012
+
+*by Titus Winters, [(titus@google.com)](mailto:titus@gmail.com) and
+Chandler Carruth [(chandlerc@google.com)](mailto:chandlerc@google.com)*
+
+*"To copy others is necessary, but to copy oneself is pathetic." - Pablo
+Picasso*
+
+Note: see also [TotW #55](/tips/55) and [TotW #77](/tips/77) for guidance
+on name counting and copies.
```

Comment:
"name counting and copies vs. moves."

Reply:
Done

Reply:
Not done?

Reply:
Ah, sorry, done now ... (Push failed before)